setup.py

```diff
@@ -1,31 +1,43 @@
-from distutils.core import setup
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# from distutils.core import setup # no support for install_requires!
+from setuptools import setup # supports install_requires!
+
+from pip.req import parse_requirements
+# See: https://stackoverflow.com/a/16624700
+install_reqs = parse_requirements('requirements.txt', session='hack')
+tests_reqs = parse_requirements('requirements-dev.txt', session='hack')
 
 setup(name='hilbert_config',
-      version='0.3.0',
+      version='0.4.0',
```

Comment:
Versions are not supposed to be incremented on a feature branch. You do this on devel, once a certain number of features/fixes has accumulated.

If you have several feature branches, they will compete about the version number. Makes no sense.

Please undo this.

Reply:
OK